OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -267,6 +279,7 @@ private Expression BindAccessor(SingleValueNode node)
                 case QueryNodeKind.SingleValueOpenPropertyAccess:
                     var openNode = node as SingleValueOpenPropertyAccessNode;
                     return Expression.Property(BindAccessor(openNode.Source), openNode.Name);
+                case QueryNodeKind.None:
```

Comment:
This case should be removed. Was a piece of code that I added there when I was trying another approach and forgot to remove.